Lib/token.py

```diff
@@ -104,13 +110,23 @@ def _main():
     prog = re.compile(
         "#define[ \t][ \t]*([A-Z0-9][A-Z0-9_]*)[ \t][ \t]*([0-9][0-9]*)",
         re.IGNORECASE)
+    comment = re.compile(
+        "^\s*/\*\s*(.+)\s*\*/\s*$",
```

Comment:
This string emits:

```
<stdin>:1: DeprecationWarning: invalid escape sequence \s
```

I suggest to always use raw strings for regular expressions.